descriptor.md

```diff
@@ -28,13 +28,14 @@ The following describe the primary set of properties that make up a _Descriptor_
   This property exists so that a client will have an expected size for the content before validating.
   If the length of the retrieved content does not match the specified length, the content SHOULD NOT be trusted.
 
-### Reserved
+- **`urls`** *array*
 
-The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
+  This OPTIONAL property specifies a list of URLs from which this object can be downloaded.
+  For an ordinary layer, this property is empty and the layer contents can be retrieved directly from the registry.
 
-- **`urls`** *array*
+### Reserved
 
-  This key is RESERVED for future versions of the specification.
+The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
```

Comment:
Can we leave this line alone?  The new line is guidance for other OCI spec implementers, not a requirement for image-tool implementers or image authors.  Using RFC 2119 wording for OCI spec authors may confuse image-tool implementers and image authors (and I'm not sure what other OCI specs would be adding a `data` field to descriptors anyway).